### ISSUE TITLE
Ignore static version in issue grouping?

### ISSUE THREAD

**Issue by peterjaap** [justbetter/magento2-sentry#63]
Everytime we do a new deploy, we suddenly see a spike in frontend errors. When I dove deeper into this, I noticed this is because after a deploy the static version signing URL is changed, and Sentry thinks it is a new issue (because new URL).

The same problem appears for different storeviews. These URLs are now seen as individual URLs while they should be grouped imho;

```
ReferenceError if(version1606832327/frontend/Elgentos/client/en_US/knockoutjs/knockout)
ReferenceError if(version1606832327/frontend/Elgentos/client/en_GB/knockoutjs/knockout)
ReferenceError if(version1606832327/frontend/Elgentos/client/nl_NL/knockoutjs/knockout)
ReferenceError if(version1606832327/frontend/Elgentos/client/de_DE/knockoutjs/knockout)

ReferenceError if(version1606832123/frontend/Elgentos/client/en_US/knockoutjs/knockout)
ReferenceError if(version1606832123/frontend/Elgentos/client/en_GB/knockoutjs/knockout)
ReferenceError if(version1606832123/frontend/Elgentos/client/nl_NL/knockoutjs/knockout)
ReferenceError if(version1606832123/frontend/Elgentos/client/de_DE/knockoutjs/knockout)
```

Is there a way to make Sentry ignore these parts of the URL when fingerprinting? I've read their docs on fingerprint but can't find anything regarding this. Or maybe we could adjust the extension so it strips out these parts (probably by introducing a new config option that is disabled by default)?

**Comment by indykoning** [comment]
We don't add any additional Javascript over setting up and configuration of Sentry so i'm afraid we will not be able to do much from this extension. It would be a great feature in general though, letting it match a pattern.

Take a look to see if there is already an issue opened on https://github.com/getsentry/sentry-javascript and if not open one.

The alternative is disabling asset signing but that is a bad alternative, since browser cache will remember the old scripts after an update.

**Comment by peterjaap** [comment]
SImilar issue; https://github.com/getsentry/sentry-javascript/issues/112

**Comment by peterjaap** [comment]
As far as I can see, we should be able to modify the event with a beforeSend hook; https://docs.sentry.io/platforms/javascript/configuration/filtering/

**Comment by peterjaap** [comment]
Here's whats inside the `event` object in `beforeSend`:

```
{
   "exception":{
      "values":[
         {
            "type":"ReferenceError",
            "value":"Unable to process binding \"if: function(){return isMsrpApplicable($row()) }\"\nMessage: isMsrpApplicable is not defined",
            "stacktrace":{
               "frames":[
                  {
                     "colno":30,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"Module.<anonymous>",
                     "in_app":true,
                     "lineno":991
                  },
                  {
                     "colno":26,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"Module.init",
                     "in_app":true,
                     "lineno":774
                  },
                  {
                     "colno":22,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"Module.enable",
                     "in_app":true,
                     "lineno":1143
                  },
                  {
                     "colno":30,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"Module.check",
                     "in_app":true,
                     "lineno":917
                  },
                  {
                     "colno":17,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"Module.emit",
                     "in_app":true,
                     "lineno":1155
                  },
                  {
                     "colno":31,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"each",
                     "in_app":true,
                     "lineno":57
                  },
                  {
                     "colno":21,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"?",
                     "in_app":true,
                     "lineno":1156
                  },
                  {
                     "colno":23,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"?",
                     "in_app":true,
                     "lineno":132
                  },
                  {
                     "colno":34,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"Module.<anonymous>",
                     "in_app":true,
                     "lineno":1113
                  },
                  {
                     "colno":51,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"Module.check",
                     "in_app":true,
                     "lineno":866
                  },
                  {
                     "colno":31,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/mage/requirejs/resolver.js",
                     "function":"Object.context.execCb",
                     "in_app":true,
                     "lineno":145
                  },
                  {
                     "colno":33,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/requirejs/require.js",
                     "function":"Object.execCb",
                     "in_app":true,
                     "lineno":1650
                  },
                  {
                     "colno":25,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Ui/js/lib/knockout/template/loader.js",
                     "function":"?",
                     "in_app":true,
                     "lineno":95
                  },
                  {
                     "colno":37,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/jquery.js",
                     "function":"Object.deferred.<computed> [as resolve]",
                     "in_app":true,
                     "lineno":3467
                  },
                  {
                     "colno":8,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/jquery.js",
                     "function":"Object.fireWith [as resolveWith]",
                     "in_app":true,
                     "lineno":3368
                  },
                  {
                     "colno":32,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/jquery.js",
                     "function":"fire",
                     "in_app":true,
                     "lineno":3238
                  },
                  {
                     "colno":42,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/jquery.js",
                     "function":"Object.<anonymous>",
                     "in_app":true,
                     "lineno":3424
                  },
                  {
                     "colno":8,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/jquery.js",
                     "function":"Object.fireWith [as resolveWith]",
                     "in_app":true,
                     "lineno":3368
                  },
                  {
                     "colno":32,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/jquery.js",
                     "function":"fire",
                     "in_app":true,
                     "lineno":3238
                  },
                  {
                     "colno":34,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/jquery.js",
                     "function":"Object.<anonymous>",
                     "in_app":true,
                     "lineno":3417
                  },
                  {
                     "colno":28,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Ui/js/lib/knockout/template/engine.js",
                     "function":"Object.<anonymous>",
                     "in_app":true,
                     "lineno":197
                  },
                  {
                     "colno":28,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"UiClass.observable [as nodes]",
                     "in_app":true,
                     "lineno":1504
                  },
                  {
                     "colno":61,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Function.valueHasMutated",
                     "in_app":true,
                     "lineno":1538
                  },
                  {
                     "colno":38,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Function.notifySubscribers",
                     "in_app":true,
                     "lineno":1320
                  },
                  {
                     "colno":32,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Function.evaluatePossiblyAsync",
                     "in_app":true,
                     "lineno":2066
                  },
                  {
                     "colno":28,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Function.evaluateImmediate",
                     "in_app":true,
                     "lineno":2101
                  },
                  {
                     "colno":29,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Function.evaluateImmediate_CallReadWithDependencyDetection",
                     "in_app":true,
                     "lineno":2140
                  },
                  {
                     "colno":103,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Function.evaluateImmediate_CallReadThenEndDependencyDetection",
                     "in_app":true,
                     "lineno":2173
                  },
                  {
                     "colno":46,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"ko.dependentObservable.disposeWhen",
                     "in_app":true,
                     "lineno":5381
                  },
                  {
                     "colno":13,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"executeTemplate",
                     "in_app":true,
                     "lineno":5339
                  },
                  {
                     "colno":13,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"activateBindingsOnContinuousNodeArray",
                     "in_app":true,
                     "lineno":5291
                  },
                  {
                     "colno":13,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"invokeForEachNodeInContinuousRange",
                     "in_app":true,
                     "lineno":5244
                  },
                  {
                     "colno":24,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"?",
                     "in_app":true,
                     "lineno":5293
                  },
                  {
                     "colno":32,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/mage/utils/wrapper.js",
                     "function":"Object.applyBindings",
                     "in_app":true,
                     "lineno":78
                  },
                  {
                     "colno":26,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Ui/js/lib/knockout/extender/bound-nodes.js",
                     "function":"Object.applyBindings",
                     "in_app":true,
                     "lineno":120
                  },
                  {
                     "colno":35,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/mage/utils/wrapper.js",
                     "function":"_super",
                     "in_app":true,
                     "lineno":73
                  },
                  {
                     "colno":9,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Object.ko.applyBindings",
                     "in_app":true,
                     "lineno":3434
                  },
                  {
                     "colno":37,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"applyBindingsToNodeAndDescendantsInternal",
                     "in_app":true,
                     "lineno":3223
                  },
                  {
                     "colno":22,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"applyBindingsToNodeInternal",
                     "in_app":true,
                     "lineno":3343
                  },
                  {
                     "colno":17,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Object.arrayForEach",
                     "in_app":true,
                     "lineno":159
                  },
                  {
                     "colno":48,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"?",
                     "in_app":true,
                     "lineno":3357
                  },
                  {
                     "colno":33,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Object.ignore",
                     "in_app":true,
                     "lineno":1470
                  },
                  {
                     "colno":46,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"?",
                     "in_app":true,
                     "lineno":3358
                  },
                  {
                     "colno":16,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"init",
                     "in_app":true,
                     "lineno":4380
                  },
                  {
                     "colno":28,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Object.ko.computed.ko.dependentObservable",
                     "in_app":true,
                     "lineno":1954
                  },
                  {
                     "colno":28,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Function.evaluateImmediate",
                     "in_app":true,
                     "lineno":2101
                  },
                  {
                     "colno":29,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Function.evaluateImmediate_CallReadWithDependencyDetection",
                     "in_app":true,
                     "lineno":2140
                  },
                  {
                     "colno":103,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"Function.evaluateImmediate_CallReadThenEndDependencyDetection",
                     "in_app":true,
                     "lineno":2173
                  },
                  {
                     "colno":32,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"ko.computed.disposeWhenNodeIsRemoved",
                     "in_app":true,
                     "lineno":4381
                  },
                  {
                     "colno":16,
                     "filename":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/knockoutjs/knockout.js",
                     "function":"if",
                     "in_app":true,
                     "lineno":2982
                  }
               ]
            },
            "mechanism":{
               "handled":true,
               "type":"instrument",
               "data":{
                  "function":"onreadystatechange",
                  "handler":"<anonymous>"
               }
            }
         }
      ]
   },
   "level":"error",
   "event_id":"b7db42aad67f4ff1b0444ad174d32257",
   "platform":"javascript",
   "sdk":{
      "name":"sentry.javascript.browser",
      "packages":[
         {
            "name":"npm:@sentry/browser",
            "version":"5.28.0"
         }
      ],
      "version":"5.28.0",
      "integrations":[
         "InboundFilters",
         "FunctionToString",
         "TryCatch",
         "Breadcrumbs",
         "GlobalHandlers",
         "LinkedErrors",
         "UserAgent"
      ]
   },
   "timestamp":1610007133.215,
   "environment":"",
   "release":"1607416581",
   "breadcrumbs":[
      {
         "timestamp":1610007124.297,
         "category":"ui.click",
         "message":"span#algolia-autocomplete-listbox-0.aa-dropdown-menu"
      },
      {
         "timestamp":1610007124.517,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/js-translation.json",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007124.852,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Ui/templates/modal/modal-popup.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007124.86,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Ui/templates/modal/modal-slide.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007124.86,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Ui/templates/modal/modal-custom.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007124.951,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"/static/deployed_version.txt",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007124.956,
         "category":"console",
         "data":{
            "arguments":[
               "Do not clear localStorage. Everything is up-to-date."
            ],
            "logger":"console"
         },
         "level":"log",
         "message":"Do not clear localStorage. Everything is up-to-date."
      },
      {
         "timestamp":1610007125.721,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Ui/templates/tooltip/tooltip.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007126.414,
         "category":"ui.click",
         "message":"div.section-item-title.nav-sections-item-title"
      },
      {
         "timestamp":1610007127.034,
         "category":"ui.click",
         "message":"div.product-reviews-summary.empty > div.reviews-actions > a.action.add"
      },
      {
         "timestamp":1610007127.315,
         "category":"ui.click",
         "message":"body.catalog-product-view.product-act-alignment-tool-clutch-90-91-b16a1-engines.categorypath-honda-aandrijving-koppelingssets.category-clutch-kits.page-layout-1column.ajax-loading"
      },
      {
         "timestamp":1610007127.48,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/en/page_cache/block/render/id/39571/?blocks=%5B%22captcha%22%5D&handles=%5B%22default%22%2C%22catalog_product_view%22%2C%22catalog_product_view_type_simple%22%2C%22catalog_product_view_id_39571%22%2C%22catalog_product_view_sku_ACT-AT87%22%2C%22algolia_search_handle%22%2C%22algolia_search_handle_with_topsearch%22%5D&originalRequest=%7B%22route%22%3A%22catalog%22%2C%22controller%22%3A%22product%22%2C%22action%22%3A%22view%22%2C%22uri%22%3A%22%2Fen%2Fact-alignment-tool-clutch-90-91-b16a1-engines%22%7D&version=updated-5fd08f9a1c10a7.98310764",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007127.707,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Ui/templates/block-loader.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007127.713,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/en/review/product/listAjax/id/39571/",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007127.927,
         "category":"console",
         "data":{
            "arguments":[
               "Fallback to JQueryUI Compat activated. Your store is missing a dependency for a jQueryUI widget. Identifying and addressing the dependency will drastically improve the performance of your site."
            ],
            "logger":"console"
         },
         "level":"warning",
         "message":"Fallback to JQueryUI Compat activated. Your store is missing a dependency for a jQueryUI widget. Identifying and addressing the dependency will drastically improve the performance of your site."
      },
      {
         "timestamp":1610007128.421,
         "category":"xhr",
         "data":{
            "method":"POST",
            "url":"https://www.google-analytics.com/j/collect?v=1&_v=j87&aip=1&a=1148956728&t=pageview&_s=1&dl=https%3A%2F%2Fwww.client.com%2Fen%2Fact-alignment-tool-clutch-90-91-b16a1-engines&ul=en-us&de=UTF-8&dt=ACT%20alignment%20tool%20clutch%20(90-91%20B16A1%20engines)%20%7C%20ACT-AT87%20%7C%20client.COM%20%7C%20client.COM&sd=24-bit&sr=2560x1440&vp=2545x892&je=0&_u=QACAAEABAAAAAC~&jid=&gjid=&cid=647503526.1592294959&tid=UA-8069009-15&_gid=1267311349.1610006814&_slc=1&z=537004084",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007128.455,
         "category":"ui.click",
         "message":"header.page-header > div.header.content > span.action.nav-toggle"
      },
      {
         "timestamp":1610007129.461,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/mage/gallery/gallery.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007129.881,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_InstantPurchase/template/confirmation.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007129.942,
         "category":"ui.click",
         "message":"div.columns > div.column.main > div.product.media > div.gallery-placeholder"
      },
      {
         "timestamp":1610007130.708,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/en/rest/en/V1/products-render-info?searchCriteria%5BfilterGroups%5D%5B0%5D%5Bfilters%5D%5B0%5D%5Bfield%5D=entity_id&searchCriteria%5BfilterGroups%5D%5B0%5D%5Bfilters%5D%5B0%5D%5Bvalue%5D=35257%2C42187%2C44106&searchCriteria%5BfilterGroups%5D%5B0%5D%5Bfilters%5D%5B0%5D%5Bcondition_type%5D=in&store_id=2&currency_code=EUR&_=1610007124053",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007130.726,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Ui/templates/collection.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007130.815,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_InstantPurchase/template/instant-purchase.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007132.073,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Catalog/template/product/list/listing.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007132.316,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Customer/template/authentication-popup.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007132.673,
         "category":"ui.click",
         "message":"div.modal-inner-wrap > header.modal-header > button.action-close[type=\"button\"]"
      },
      {
         "timestamp":1610007133.018,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Catalog/template/product/list/columns/image_with_borders.html",
            "status_code":200
         },
         "type":"http"
      },
      {
         "timestamp":1610007133.141,
         "category":"xhr",
         "data":{
            "method":"GET",
            "url":"https://www.client.com/static/version1607416581/frontend/Elgentos/client/en_GB/Magento_Msrp/template/product/price/price_box.html",
            "status_code":200
         },
         "type":"http"
      }
   ],
   "request":{
      "url":"https://www.client.com/en/act-alignment-tool-clutch-90-91-b16a1-engines",
      "headers":{
         "Referer":"https://www.client.com/en/honda-drivetrain/clutch-kits",
         "User-Agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.101 Safari/537.36"
      }
   },
   "extra":{
      "arguments":[
         {
            "type":"readystatechange",
            "target":"[object XMLHttpRequest]",
            "currentTarget":"[object XMLHttpRequest]",
            "isTrusted":"[object Event]"
         }
      ]
   }
}
```

**Comment by peterjaap** [comment]
Score!

![image](https://user-images.githubusercontent.com/431360/103870372-e7a70d00-50cb-11eb-8f91-4b81b2d687af.png)

**Comment by peterjaap** [comment]
I'll test this in production for of our stores, will report back.

**Comment by peterjaap** [comment]
I'm thinking we should make this a configuration setting (which defaults to disabled in order to stay backwards compatible). We could add an option to strip out the store code as well.

**Comment by indykoning** [comment]
That's great that we can fix it from our side!
I do agree that we may want to have it as a configuration option for whoever is dependent on de version being pushed as well.

Yes, either another option to remove the language as well. Or an input field to define regexes with patterns to remove items from the url. However maybe adding a field for regexes will be too complicated an cluttered to manage from the backend.

 Anyhow i'm looking forward to your PR 😄

**Comment by peterjaap** [comment]
@indykoning first stab at this; https://github.com/justbetter/magento2-sentry/pull/73

I've added a group and two settings (one if the store is in single store mode, because stripping the store code wouldn't make sense in that case);

![image](https://user-images.githubusercontent.com/431360/106741403-e9f17e00-661b-11eb-8763-732a4f97dd20.png)

**Comment by indykoning** [comment]
It looked good to me and is available as version 2.5.3 🚀 
I agree with allowing people to enable/disable it since if we were to just change it it could cause some confusion.
And of course not all shops use the store code in the url.
